BasicAPI6/LinkedTables.java

```diff
@@ -0,0 +1,61 @@
+package BasicAPI6;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+import java.util.TreeMap;
+
+// класс для работы с таблицами связанными с экземплярами Notebook
+public class LinkedTables {
+    private HashMap<String, Map<Integer, String>> tables; // связь между ключами Notebook и связанными таблицами
+
+    LinkedTables(){
+        tables = new HashMap<>();
+    }
+
+    public boolean set(String key, Map<Integer, String> object){
+        if (Notebook.IDKEYS.contains(key)){// если вводимый ключ соответствует связанной таблице
+            this.tables.put(key, object);
+            return true;
+        } else {
+            return false;
+        }
+    }
+
+    public Map<Integer, String> get(String key){
+        if (this.tables.containsKey(key)){
+            return this.tables.get(key);
+        } else {
+            return null;
+        }
+    }
+
+    public static Map<Integer, String> loadTable(String fileName){
```

Comment:
This method should be in another class, how it is related to LinkedTables?

Reply:
Linked Tables contains all fields and methods necessary to work with data in linked tables, including a static method to load data from csv files.